draft-ietf-dnsop-svcb-https.md

```diff
@@ -867,7 +867,30 @@ compliant recursive resolvers.  When SvcDomainName is ".", server operators
 SHOULD NOT include these hints, because they are unlikely to convey any
 performance benefit.
 
+## "mandatory" {#mandatory}
 
+In a ServiceForm RR, a SvcParamKey is considered "essential" if the RR will not
+function correctly for clients that ignore this SvcParamKey.  Each SVCB
+protocol mapping SHOULD specify a default set of essential keys.  The
+SvcParamKey "mandatory" is used to indicate the set of essential keys for this
+RR (overriding the default set).
```

Comment:
Overriding or adding to the default set?  The later seems more usable?

Reply:
Overriding seems more powerful, since it allows making default-essential keys optional.  Given that this feature should be very rare, I figured usability was not too important.

Then again, maybe clients have to have an understanding of all the default-essential keys for a scheme anyway, so I guess we can just make it additive.

Reply:
Changed: I made it additional instead.